### ISSUE TITLE
Commit 331ae036bed29021d3326c79e2339a7efd95a4b6 breaks testing

### ISSUE THREAD

**Issue by DSANTIMORE** [dotnet/dotnet-docker#1034]
331ae036bed29021d3326c79e2339a7efd95a4b6 appears to break the test output file name conventions that used to exist (this is mentioned https://github.com/Microsoft/vstest/issues/1876#issuecomment-470318756 as well).

Is there a way to target the before this commit (so, dotnet test 15.9) using tagging of FROM microsoft/dotnet:2.1-sdk

Many thanks for the repy.

**Comment by mthalman** [comment]
@DSANTIMORE - You can unblock yourself while we investigate this by referencing the previous version of the image in this way:
`FROM mcr.microsoft.com/dotnet/core/sdk:2.1.505`

**Comment by DSANTIMORE** [comment]
Thank you for the quick reply. And we've found a way to use wild cards correctly to handle it more elegantly, as well.

**Comment by mthalman** [comment]
@DSANTIMORE - Can you provide details on the actual and expected behaviors you're seeing?  From looking at Microsoft/vstest#1876, it looks like the only work that has been done so far is to add a timestamp at the end of the user-provided log file name.  You're saying that after making use of the latest changes from [331ae03](https://github.com/dotnet/dotnet-docker/commit/331ae036bed29021d3326c79e2339a7efd95a4b6), you're no longer seeing the timestamp automatically added at the end log file name?

**Comment by DSANTIMORE** [comment]
@mthalman , my apologies, I didn't write up a very good summary. The breaking behavior was the addition of the timestamp.

I think your tag note above is a sufficient answer for me; basically, the change of the file name include the timestamp (which is a change from v15.9 --> v16.0) was a pretty serious breaking change that ended pretty much all of our builds today. We took some time to sort out that we could make it work with the wildcard syntax with another library, trx2junit; before that, we did not believe we could find a way forward without pegging the tag you referenced above.

I'm in no position to describe the timestamp as unexpected behavior; only it is definitely breaking behavior and we can adapt.
